heron/scheduler-core/src/java/com/twitter/heron/scheduler/TopologyRuntimeManagementException.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler;
+
+/**
+ * Thrown to indicate that an error occurred while managing topology runtime
+ */
+public class TopologyRuntimeManagementException extends RuntimeException {
```

Comment:
then accordingly, `TopologySubmissionException` should be `SubmissionException`?